### ISSUE TITLE
Resnet with Group Normalization pretrained weight sharing?

### ISSUE THREAD

**Issue by fjchange** [ruotianluo/pytorch-resnet#6]
When I run the convert_gn.py as readme,  there is file, "R-50-GN.pkl" not found but no download link offered. Can you share the pretrained model? Very thanks.

**Comment by ruotianluo** [comment]
https://dl.fbaipublicfiles.com/detectron/ImageNetPretrained/47261647/R-50-GN.pkl

**Comment by fjchange** [comment]
Thanks, I find it just after opened the issue.
